pytorch_lightning/accelerators/gpu.py

```diff
@@ -75,6 +76,16 @@ def get_device_stats(self, device: _DEVICE) -> dict[str, Any]:
             return torch.cuda.memory_stats(device)
         return get_nvidia_gpu_stats(device)
 
+    @staticmethod
+    def parse_devices(devices: int | str | list[int]) -> list[int] | None:
+        """Accelerator Parsing logic."""
+        return device_parser.parse_gpu_ids(devices)
+
+    @staticmethod
+    def get_parallel_devices(devices: list[int]) -> list[torch.device]:
```

Comment:
again, is this typing compatible with 3.6 and/or 3.7?

Reply:
Fixed!